### PR TITLE
Feature/q button

### PR DESCRIPTION
**Pull request by MrZyr0**

### CODE REVIEW

**Review by Laroche-Nicolas — APPROVED** [review]

File: src/components/QLoader/QLoader.vue [hunk 1]
```diff
@@ -1,17 +1,36 @@
 <template>
-  <span class="q-loader spinner" :style="`border-top-color: ${borderTopColor}`" />
+  <span class="q-loader spinner" :style="`border-color: ${bgColor}; border-top-color: ${accentColor}; height: ${size}px; width: ${size}px; border-width: ${borderWidth}px`" />
```
Comment (MaxenceMottard) [inline src/components/QLoader/QLoader.vue:2]: Please change like this : https://github.com/quiches-group/quichtify/blob/master/src/components/QTabs/QTabs.vue

**Review by MaxenceMottard — CHANGES_REQUESTED** [review]

File: src/components/QLoader/QLoader.vue (hunk 1 above)
Comment (MrZyr0) [reply src/components/QLoader/QLoader.vue:2]: Impossible to make it work so I leave it like that...

**Review by nicolasbarb — APPROVED** [review]

File: src/components/QButton/QButton.stories.js [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+    },
+    disabled: {
+      control: {
+        type: 'boolean',
+      },
+    },
+    // FIXME
+    // 'v-on:click': {
+    //   action: 'onClick',
+    // },
```
Comment (MaxenceMottard) [inline src/components/QButton/QButton.stories.js:64]: Remove or fix ?

**Review by MaxenceMottard — APPROVED** [review]
One thing is sure, it doesn't make you want to try storybook at all

File: src/components/QButton/QButton.stories.js (hunk 2 above)
Comment (MrZyr0) [reply src/components/QButton/QButton.stories.js]: After many tries I don't find how to fix it... 😢

**Pull request merged**